packages/@aws-cdk/aws-lambda-nodejs/README.md

```diff
@@ -101,5 +101,24 @@ new lambda.NodejsFunction(this, 'my-handler', {
 
 The modules listed in `nodeModules` must be present in the `package.json`'s dependencies. The
 same version will be used for installation. If a lock file is detected (`package-lock.json` or
-`yarn.lock`) it will be used along with the right installer (`npm` or `yarn`). The modules are
-installed in a [Lambda compatible Docker container](https://hub.docker.com/r/amazon/aws-sam-cli-build-image-nodejs12.x).
+`yarn.lock`) it will be used along with the right installer (`npm` or `yarn`).
+
+
+### Local bundling
+If Parcel v2 is available it will be used to bundle your code in your environment. Otherwise,
+bundling will happen in a [Lambda compatible Docker container](https://hub.docker.com/r/amazon/aws-sam-cli-build-image-nodejs12.x).
+
+Parcel v2 can be installed with:
+
+```bash
+$ npm install --save-dev parcel@next
+```
+
+OR
+
+```bash
+$ yarn add --dev @parcel@next
+```
+
+To force bundling in a Docker container, set the `forceDockerBundling` prop to `true`. This is useful
+when installing node modules with native dependencies.
```

Comment:
The sentence "This is useful when installing node modules with native dependencies" is a bit cryptic. Elaborate a bit or provide an example.